prac_05/wimbledon.py

```diff
@@ -0,0 +1,51 @@
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual:  60 minutes
+"""
+FILENAME = "wimbledon.csv"
+INDEX_COUNTRY = 1
+INDEX_CHAMPION = 2
+
+
+def main():
+    """Process data from a file and format accordingly."""
+    records = get_records(FILENAME)
+    champion_to_count, countries = process_records(records)
+    display_results(champion_to_count, countries)
+
+
+def process_records(records):
+    """Create a count of players and championships won."""
+    champion_to_count = {}
+    countries = set()
+    for record in records:
+        countries.add(record[INDEX_COUNTRY])
+        try:
+            champion_to_count[record[INDEX_CHAMPION]] += 1
```

Comment:
Would lines 25 & 27 work better using champion_to_count.get (similar to how you used it in word_occurrences.py)?